components/eam/cime_config/config_component.xml

```diff
@@ -128,7 +128,9 @@
       <value compset="GEOS_EAM"      >-nlev 56</value>
       <value compset="_EAM.*_BGC%B"   >-co2_cycle</value>
       <value compset="EAM%SCAM"   >-scam</value>
-      <value compset="AR97_EAM"   >-phys cam5 -scam -nlev 72 -clubb_sgs -chem linoz_mam4_resus_mom_soag -rain_evap_to_coarse_aero -bc_dep_to_snow_updates</value>
+
+      <value compset="AR97_EAM%SCAM" >-phys cam5 -scam -nlev 72 -shoc_sgs -microphys p3 -rad rrtmgp -chem linoz_mam4_resus_mom_soag -rain_evap_to_coarse_aero -bc_dep_to_snow_updates</value>
```

Comment:
Seeing the errors reported on slack caused by the recent upstream merge, `-phys cam5` might need to be replaced with `-phys default`.